server/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const db = require('../db/seed.js');
+
+let app = express();
+app.use(express.static(__dirname + '/../client/dist'));
+
+app.get('/streamers', function (req, res) {
+    console.log('server is getting data');
+    db.stream.find((err, data) => {
+        if (err) {
+            console.log(err);
```

Comment:
In case of an error, shouldn't we still respond to the request?

```suggestion
        if (err) {
            console.log(err) //for troubleshooting purposes
            res.status(500).send('something went wrong!')
```